src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -63,6 +65,11 @@ export default App.extend({
       storedState.lastSelectedIndex = null;
     }
 
+    if (!storedState) {
+      this.setState({ id: this.worklistId, filters: { states: initialStatesStatusObject } });
+      return;
+    }
```

Comment:
If the user has no local storage data for the list, this sets the default states to all be selected.

Ideally, this would be done directly in the `worklist_state.js` file, but I need access to the `this.worklistId`. Which is set after the state is initially created.